src/pages/iou/IOUCurrencySelection.js

```diff
@@ -126,8 +126,11 @@ function IOUCurrencySelection(props) {
             };
         });
 
-        const searchRegex = new RegExp(Str.escapeForRegExp(searchValue.trim()), 'i');
-        const filteredCurrencies = _.filter(currencyOptions, (currencyOption) => searchRegex.test(currencyOption.text) || searchRegex.test(currencyOption.currencyName));
+        const searchRegex = new RegExp(Str.escapeForRegExp(searchValue.trim().replace(/\s/g, ' ')), 'i');
```

Comment:
@ZhenjaHorbach Kindly add regex as a CONST here, if not already present.

https://github.com/Expensify/App/blob/8088a7bb669162e9004ffd7ad96f0b44ae870742/src/CONST.ts#L1313